### PR TITLE
Planet Minecraft

### PR DESCRIPTION
**Pull request by night-lake**
## 🎉 Theme for [Planet Minecraft](https://planetminecraft.com)🎉

Planet Minecraft is a non-official website for Minecraft players to share, download and interact with resources such as skins, mods, resource packs, etc.

## 💬 Additional Comments 💬

> **Note**
> This theme is still in progress, if you have any comments or find something unthemed, please tell me in this PR! Thus, the catwalk will be generated when the theme is ready for merging.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/src/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [catwalk](https://github.com/catppuccin/toolbox#-catwalk).

### CODE REVIEW

**Comment by night-lake** [comment]
Known Issues:
- [ ] These icons aren't properly visible in Latte (should be Text)
	- image
- [ ] The account dropdown isn't properly themed
	- image
- [ ] The Time Machine menu isn't properly themed, as well as basically the entire thing
	- image

File: template/catppuccin.user.css
Comment (isabelroses) [inline template/catppuccin.user.css:1]: can you revert the changes you made to this file please.

File: styles/planet-minecraft/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  @macchiato: { @rosewater: #f4dbd6; @flamingo: #f0c6c6; @pink: #f5bde6; @mauve: #c6a0f6; @red: #ed8796; @maroon: #ee99a0; @peach: #f5a97f; @yellow: #eed49f; @green: #a6da95; @teal: #8bd5ca; @sky: #91d7e3; @sapphire: #7dc4e4; @blue: #8aadf4; @lavender: #b7bdf8; @text: #cad3f5; @subtext1: #b8c0e0; @subtext0: #a5adcb; @overlay2: #939ab7; @overlay1: #8087a2; @overlay0: #6e738d; @surface2: #5b6078; @surface1: #494d64; @surface0: #363a4f; @base: #24273a; @mantle: #1e2030; @crust: #181926; }
+  @mocha:     { @rosewater: #f5e0dc; @flamingo: #f2cdcd; @pink: #f5c2e7; @mauve: #cba6f7; @red: #f38ba8; @maroon: #eba0ac; @peach: #fab387; @yellow: #f9e2af; @green: #a6e3a1; @teal: #94e2d5; @sky: #89dceb; @sapphire: #74c7ec; @blue: #89b4fa; @lavender: #b4befe; @text: #cdd6f4; @subtext1: #bac2de; @subtext0: #a6adc8; @overlay2: #9399b2; @overlay1: #7f849c; @overlay0: #6c7086; @surface2: #585b70; @surface1: #45475a; @surface0: #313244; @base: #1e1e2e; @mantle: #181825; @crust: #11111b; }
+}
+
+
+/*  Userstyle as Mixin
+ *  This takes the following Parameters:
+ *  @@lookup – which is the desired Flavor
+ *  @@accent – which is the desired Accent Color */
```
Comment (isabelroses) [inline styles/planet-minecraft/catppuccin.user.css:34]: You don't need to keep all these comments in.

**Review by isabelroses — CHANGES_REQUESTED** [review]
Some things still are not themed such as 
image
which is themed using the variable `--graphic-bg`.
as well as some button. 
image

Perhaps checkout the root variables on the page as I have found a number of unthemed variables, perhaps they are not need ones but it could be done to prevent future issues.

**Comment by night-lake** [comment]
just for confirmation, im coloring the panes in the correct order right (mantle for lowest, base for highest)

**Comment by isabelroses** [comment]
> just for confirmation, im coloring the panes in the correct order right (mantle for lowest, base for highest)

When working inwards try make the colors become lighter going from crust to mantle to base, but keep base where the main focus is. Perhaps where any cards are you could use something like surface0 etc.

**Comment by night-lake** [comment]
i still have to fix these buttons
image

**Comment by isabelroses** [comment]
Just some things to point out.
1. The text chat has unthemed borders: 
image
2. group notifications are only partly themed
image
3. Buttons upon a hover lose there darker text.

**Review by isabelroses — APPROVED** [review]

**Pull request merged**